baseline.yaml

```diff
@@ -649,6 +649,26 @@ criteria:
     security_insights_value: # TODO
     scorecard_probe: # TODO
 
+  - id: OSPS-DO-17   
+    maturity_level: 1
```

Comment:
If this is a maturity level 1 requirement, it feels like we should be more descriptive of how high this wall is.  For example, is having 2 paragraphs under a heading in `SECURITY.md` sufficient, or is a stand-alone document required?  Are there specific frameworks that should be referenced, or threat models documented?

Reply:
I agree. This needs to be explained in much more detail, particularly if it's level 1. Depending on what we mean by "risk assessment", I'm inclined to think this is maybe a level 2 requirement?

Reply:
concur with level 2 - but we should double check against what other criteria would be considered part of a risk assessment by the project (threat model, security analysis, etc.) to ensure we're not setting this too low if components that contribute are at level 3